lib/storage/acl.js

```diff
@@ -67,17 +69,138 @@ var util = require('../common/util.js');
  * @alias module:storage/acl
  */
 function Acl(options) {
+  AclRoleAccessorMethods.call(this);
+
   this.makeReq = options.makeReq;
   this.pathPrefix = options.pathPrefix;
 }
 
 /**
+ * A convenience object of methods to add or delete ACL permissions from an
```

Comment:
I'm assuming this is public documentation so perhaps we should clarify the role of these methods is for owners. Example:  "A convenience object of methods to add and delete **owner** ACL permissions from different entities."